lib/thrift/parser/file_group.ex

```diff
@@ -87,28 +87,29 @@ defmodule Thrift.Parser.FileGroup do
     |> Enum.find(path, &File.exists?/1)
   end
 
-  @spec update_resolutions(t) :: t
-  def update_resolutions(file_group) do
+  @spec set_current_module(t, atom) :: t
+  def set_current_module(file_group, module) do
     # since in a file, we can refer to things defined in that file in a non-qualified
     # way, we add unqualified names to the resolutions map.
-    to_update = file_group.resolutions
-    |> Enum.map(fn {name, v} ->
-      case String.split(Atom.to_string(name), ".") do
-        [_module, enum_name, value_name] ->
-          {:"#{enum_name}.#{value_name}", v}
-        [_initial_module, rest] ->
-          {:"#{rest}", v}
-      end
+
+    current_module = Atom.to_string(module)
```

Comment:
This is one of two cases; in `generator.ex`, we use the schema's module which is an atom. I also felt that the module make sense as an attribute since elixir modules are just atoms.

Also, think of this as step one of two. I think moving all of this to Schema will make so much more sense.